controllers/provider-openstack/charts/internal/cloud-provider-config/templates/cloud-provider-config.yaml

```diff
@@ -15,7 +15,12 @@ data:
     lb-version=v2
```

Comment:
Move all the parts that are common in the cloud provider configs for KCM and CCM into a template (like it's done for Azure: https://github.com/gardener/gardener-extensions/tree/master/controllers/provider-azure/charts/internal/cloud-provider-config/templates)